pkg/loadbalancer/loadbalancer.go

```diff
@@ -425,30 +430,23 @@ func (b *Backend) GetBackendModel() *models.BackendAddress {
 	}
 }
 
-// String returns the L3n4Addr in the "IPv4:Port[/Scope]" format for IPv4 and
-// "[IPv6]:Port[/Scope]" format for IPv6.
+// String returns the L3n4Addr in the "IPv4:Port/Protocol[/Scope]" format for IPv4 and
+// "[IPv6]:Port/Protocol[/Scope]" format for IPv6.
 func (a *L3n4Addr) String() string {
-	var scope string
-	if a.Scope == ScopeInternal {
-		scope = "/i"
-	}
-	if a.IsIPv6() {
-		return fmt.Sprintf("[%s]:%d%s", a.IP.String(), a.Port, scope)
+	var (
+		scope string
+		proto string
+	)
+	if a.Protocol != NONE {
+		proto = fmt.Sprintf("/%s", a.Protocol)
 	}
```

Comment:
The function documentation does not seem to be entirely truthful. If the protocol is "NONE" the String returned won't contain the `/Protocol` section.


What's wrong on also printing the protocol if it's `NONE`

Reply:
Yeah we should also add an `/ANY` or such to denote it's both TCP/UDP. Do we log errors on SCTP services to the user?

Reply:
No.

Reply:
Should we?